api/src/test/java/org/pmiops/workbench/disk/DiskServiceTest.java

```diff
@@ -70,13 +71,12 @@ public void test_findByWorkspaceNamespace() {
             user,
             AppType.RSTUDIO);
     LeonardoListPersistentDiskResponse thirdLPDR =
-        createLeonardoListPersistentDiskResponse(
+        createLeonardoListRuntimePDResponse(
```

Comment:
leaving a note here: the reason for this change is to show that it works for the multiple-runtime-disks edge case relating to the bug of March/April